RecoParticleFlow/PFClusterProducer/python/particleFlowClusterHBHE_cfi.py

```diff
@@ -86,3 +86,159 @@
     energyCorrector = cms.PSet()
 )
 
+#####
+
+seedFinderThresholdsByDetector2017 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+
+seedFinderThresholdsByDetector2018 = particleFlowClusterHBHE.seedFinder.thresholdsByDetector
+seedFinderThresholdsByDetector2018 = cms.VPSet(
```

Comment:
.. better confirm that these actually mean new sets are created. Otherwise the same particleFlowClusterHBHE.seedFinder.thresholdsByDetector is modified several times